configure.ac

```diff
@@ -4,13 +4,14 @@ define(_CLIENT_VERSION_MAJOR, 0)
 define(_CLIENT_VERSION_MINOR, 10)
 define(_CLIENT_VERSION_REVISION, 2)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_CLIENT_VERSION_IS_RELEASE, true)
```

Comment:
Hmm.. I think so, yes, but I'm absolutely open to this.

The underlying idea is basically that the RC is as close to the actual release as possible, and if no new issues surface, then RC == release (except the version tag).

On the other hand, if there are indeed issues with the RC, then a version without warning may create the impression it's final.